e2e/epinio.e2e.spec.ts

```diff
@@ -0,0 +1,239 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import { ElectronApplication, BrowserContext, _electron, Page } from 'playwright';
+import { test, expect } from '@playwright/test';
+import {
+  createDefaultSettings, tearDownHelm, playwrightReportAssets, setUpHelmCustomEnv, helm, kubectl,
+} from './utils/TestUtils';
+import { NavPage } from './pages/nav-page';
+import * as childProcess from '@/utils/childProcess';
+
+let page: Page;
+
+test.describe.serial('Epinio Install Test', () => {
+  // Disabling this test for linux and windows - See https://github.com/rancher-sandbox/rancher-desktop/issues/1634
+  test.skip(os.platform().startsWith('linux') || os.platform().startsWith('win'), 'Need further investigation on Linux runner');
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+
+  test.beforeAll(async() => {
+    installEpinioCli();
+    createDefaultSettings();
+    setUpHelmCustomEnv();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        '--disable-dev-shm-usage',
+      ]
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+  });
+
+  /**
+   * helm teardown
+   * It should run outside of the electronApp.close(), just to make sure the teardown won't
+   * affect the shutdown process in case of exceptions/errors.
+   */
+  test.afterAll(tearDownHelm);
+
+  /**
+   * epinio teardown
+   * It removes epinio binary temp folder along with its binary and
+   * perform a helm uninstall epinio-installer.
+   */
+  test.afterAll(tearDownEpinio);
```

Comment:
`tearDownEpinio` needs to run before `tearDownHelm` (because we remove epinio via helm).

I'm also not sure we need doc comments for a random line of code… it's not a function/class/interface declaration.

Reply:
Done